Azure_Pipelines/scale-test/azure-pipelines.account.cd.yaml

```
@@ -0,0 +1,68 @@
+trigger:
+  branches:
+    include:
+    - release/0.1.1
+  paths:
+    include:
+    - DotNet/Account/*
+    - DotNet/Shared/*
+    exclude:
+    - '*'
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)